app/services/participants/resume/base.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "abstract_interface"
+
+module Participants
+  module Resume
+    class Base < ::Participants::Base; end
```

Comment:
Out of curiosity, why the `::Participants` here and nowhere else?

Reply:
I think if you are in the same module, you can short hand it. The `Base` in other places, really is `Participants::Resume::Base`, but since we are in `Participants::Resume` module, it's fine.

I guess `class Base < Participants::Base; end` is good enough for our needs. Not sure what `::` in front of it does. Something clever that also seems to work.